IdentityCore/src/util/NSString+MSIDExtensions.m

```diff
@@ -424,4 +251,48 @@ - (BOOL)msidIsEquivalentWithAnyAlias:(NSArray<NSString *> *)aliases
     return NO;
 }
 
+
++ (NSString *)msidStringFromOrderedSet:(NSOrderedSet *)set
+{
+    NSInteger cSet = set.count;
+    if (cSet == 0)
+    {
+        return @"";
+    }
+    
+    NSMutableString *queryString = [[set objectAtIndex:0] mutableCopy];
```

Comment:
nit: might be easier to use componentsJoinedByString here with a tradeoff of using nsarray

Reply:
good point!